src/views/BillDisplay/BillDisplay.jsx

```diff
@@ -1,61 +1,104 @@
 import React, { Component } from "react";
 import "./styles.scss";
 import Loader from "../../components/Loader";
+import SmallBillCard from "../../components/BillCard/SmallBillCard";
+import LargeBillCard from "../../components/BillCard/LargeBillCard";
+import { Nav, NavItem, NavLink } from "shards-react";
+import { FaUtensils, FaShoppingCart, FaShoppingBag, FaCar, FaBus, FaQuestion, FaSearch } from 'react-icons/fa';
+import {navItems} from "../../constants/BillDisplayNav";
 
 class BillDisplay extends Component {
   constructor(props) {
     super(props);
     this.state = {
       user: { firstName: "", lastName: "" },
+      currentActiveTab: "allBills"
     };
   }
 
   componentDidMount = async () => {
     await this.props.fetchBills();
   };
 
-  static billStatusColor = (status) => {
-    switch (status) {
-      case "OPEN":
-        return "success";
-      case "RESOLVED":
-        return "primary";
-      case "IN_PROGRESS":
-        return "warning";
-      default:
-        return "muted";
+   billIcons = category => {
+        const color = "rgba(0, 0, 0, 0.96)";
+        switch(category){
+            case "food":
+                return <FaUtensils color={color} size={24}/>;
+            case "transport":
+                return <FaCar color={color} size={24}/>;
+            case "public-transport":
+                return <FaBus color={color} size={24}/>;
+            case "grocery":
+                return <FaShoppingCart color={color} size={24}/>;
+            case "shopping":
+                return <FaShoppingBag color={color} size={24}/>;
+            default:
+                return <FaQuestion color={color} size={24}/>;   
+        };
+    };
+   
+    filterDateTime = (timestamp) => {
```

Comment:
Since this function does not rely on the class, you can set this function as a `static` function. 

Then you can call it via `this.constructor.fildterDateTime`